packages/marshal/src/encodeToCapData.js

```diff
@@ -432,16 +433,9 @@ export const makeDecodeFromCapData = ({
       return result;
     } else {
       assert(typeof jsonEncoded === 'object' && jsonEncoded !== null);
```

Comment:
Should we keep the no-symbol-keys assertion?
```suggestion
      assert(typeof jsonEncoded === 'object' && jsonEncoded !== null);
      for (const name of ownKeys(jsonEncoded)) {
        typeof name === 'string' ||
          assert.fail(X`Property ${name} of ${jsonEncoded} must be a string`);
      }
```
or in a single pass:
```suggestion
      assert(typeof jsonEncoded === 'object' && jsonEncoded !== null);
      const decodedEntries = ownKeys(jsonEncoded).map(name =>
        typeof name === 'string' ||
          assert.fail(X`Property ${name} of ${jsonEncoded} must be a string`);
        return [name, decodeFromCapData(jsonEncoded[name])];
      );
      return fromEntries(decodedEntries);
```

Reply:
Done